assets/js/googlesitekit-datastore-site.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
```

Comment:
@tofumatt Btw this should go right under "External dependencies" above, to "correctly" please the linter. I know this isn't optimal, but let's stick to what it provides us for now rather than doing this "hack" to just add the "External dependencies" somewhere before.

We should address this at some point, most likely by introducing a 4th layer like "Site Kit dependencies" (in addition to the existing "External", "WordPress", "Internal".

Reply:
I find that makes code management _very_ confusing. I don't think we should be pleasing the linter here—it's a tool we use to ensure code makes sense to us but in this case it's not working as intended. `'googlesitekit-*'` modules are internal dependencies, and marking them as anything else will confuse new developers without benefit to us. I think it's better to write the code awkwardly once (as is done here) to improve readability (especially for new contributors) than the other way around 🙂

Reply:
Okay with me, but we should really fix the linter then. Maybe we can put exceptions for `googlesitekit-*` to be treated as Internal dependencies?

Reply:
We can't currently change what dependencies are considered internal by ESLint. See https://github.com/google/site-kit-wp/issues/780#issuecomment-549815649

According to the current rule, Internal dependencies are only those that start with a `.`